src/Symfony/Component/Serializer/NameConverter/ChainNameConverter.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\NameConverter;
+
+/**
+ * ChainNameConverter allows to process a property name with multiple converters.
+ *
+ * @author Jérôme Gamez <jerome@gamez.name>
+ */
+class ChainNameConverter implements NameConverterInterface
+{
+    /**
+     * @var NameConverterInterface[]
+     */
+    protected $converters = array();
```

Comment:
Changed (I prefer `private`s, too, but I took the ChainLoader as an example - which perhaps wasn't the best idea, because it's so old already :))